lms/services/http.py

```diff
@@ -65,6 +68,12 @@ def request(
         :param schema: A schema class to use to validate the response
         :type schema: lms.validation.RequestsResponseSchema
 
+        :param oauth: Include an OAuth 2 access token in the request.
+            If oauth=True the current user's access token will be looked up in
+            the database and included in an Authorization header in the
+            request.
+        :type oauth: bool
```

Comment:
I find this :type style doc string to be a bit clunky and much prefer the type annotation on the function definition but it's also true that adding for example just this one `oauth: bool = True` annotation would stand out as the only annotation in the fuction/module/maybe even project?

Reply:
Part of the problem is that I haven't gotten used to type annotations yet so I can't easily write them.

Although I did notice that the `:type:`'s keep the type documentation close to the `:param:` documentation, rather than the `:param:` being down in the docstring and the type annotation being way up at the top.

Realistically we should probably start using type annotations as they're clearly where the industry is going